app/main/routes.py

```diff
@@ -620,7 +621,7 @@ def record(record_id: uuid.UUID):
     )
 
     try:
-        manage_static_file(file, file_extension)
+        presigned_url = create_presigned_url(file)
     except Exception as e:
```

Comment:
I think it would be worth making the error message more specific, saying its something like "Failed to create presigned url for document render non-javascript fallback" and also logging the original error message/stacktrace